packages/contentstack-import/src/import/modules/base-class.ts

```diff
@@ -306,11 +306,11 @@ export default abstract class BaseClass {
       case 'update-gfs':
         return apiData.update().then(onSuccess).catch(onReject);
       case 'create-environments':
-      // return this.stack
-      //   .environment()
-      //   .create({ environment: omit(apiData, ['uid']) as EnvironmentData })
-      //   .then(onSuccess)
-      //   .catch(onReject);
+        return this.stack
```

Comment:
Can you update the management sdk version wherever required ?

Reply:
@shafeeqd959 done